internal/pkg/cli/pipeline_init.go

```diff
@@ -193,14 +195,18 @@ func (opts *InitPipelineOpts) createPipelineManifest() (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("marshal manifest: %w", err)
 	}
-	manifestPath, err := opts.manifestWriter.WriteManifest(manifestBytes, pipelineDefaultFilename)
+	manifestPath, err := opts.manifestWriter.WriteFile(manifestBytes, pipelineDefaultFilename)
```

Comment:
Could you add a function, like we have for AppManifestFileName - PipelineFileName()?

Reply:
Sounds good, I can add it in my follow-up PR that writes the build spec!